portality/view/doaj.py

```diff
@@ -356,11 +356,12 @@ def toc_articles_legacy(identifier=None):
 def toc_articles(identifier=None):
     journal = find_toc_journal_by_identifier(identifier)
     bibjson = journal.bibjson()
+    articles_no = len(list(journal.all_articles()))
```

Comment:
This is an incredibly inefficient way to get the article count, because it transfers the full article JSON from the index. See `Journal.article_stats()` to get this information. 

https://github.com/DOAJ/doaj/blob/b4116536601cce1879a2cb791150dae83aa41fe1/portality/models/v2/journal.py#L600